.github/ISSUE_TEMPLATE/give_a_talk.yml

```
@@ -0,0 +1,43 @@
+name: Give a TTW talk 
+description: Resources and checklist for giving a talk about the Turing Way
+title: "[TALK] <title>"
+labels: [talks-and-workshops, newsletter]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This template will guide you through the steps necessary to give a successful talk about The Turing Way!
+  - type: input
+    id: date
+    attributes:
+      label: Date of talk
+      description: When will you be presenting about the Turing Way?
+      placeholder: please use the format YYYY-MM-DD
+    validations:
+      required: true
+  - type: input
+    id: talk_details
+    attributes:
+      label: Details of the talk
+      description: Where are you presenting? Please provide a link to the event if available
+  - type: checkboxes
+    id: checklist
```

```suggestion
    id: pre-talk-checklist
```

This form doesn't yet load correctly because each value of `id` must be unique